### ISSUE TITLE
Users needed to know where to click but were unsure and required assistance

### ISSUE THREAD

**Issue by joanneSouthern** [alphagov/verify-local-patterns#337]

**Comment by suryanaga** [comment]
@joanneSouthern For this one, it would be good to understand more about the users who said they'd need assistance, and what type of digital experience they had. As well as what specifically they were struggling with.

For example, if this was users who said they owned an iPad but had never used a mouse, and then struggled to use the mouse, I don't think we need to worry too much. (Although it does highlight that we need to be doing more testing with iPads to be more realistic!)

**Comment by joanneSouthern** [comment]
low IT skills, i would even say none.  They had ipads so werent used to a mouse, struggled with scrolling, clicking and moving the mouse, we had to do the mouse for them,
Even the people with IT skills said they wanted assistance, they weren't sure where to click and didnt understand what they needed to do or why they were doing it

**Comment by suryanaga** [comment]
Ok, a think there's two things rolled up here:

 1. Users who haven't used a mouse before find it hard to use a mouse. We should be testing with those users with an iPad or whatever other device they use at home.
 2. People unsure where to click. This I think is a different issue that's related to the current Northumberland styling. 

The first issue is really a question for further research, not really directly addressable until we've done iPad testing. I've documented it as a question in #390.

The second thing should definitely be documented as an issue, with the action being more work being done on the Northumberland version of the prototype. Unless there's a similar issue somewhere else, I'd reword this issue "Users need a clearer indication of where to click"

**Comment by irinapencheva** [comment]
@joanneSouthern - are you happy with the above suggestion from Sanjay?

**Comment by irinapencheva** [comment]
also this is linked to #347 so if you're happy we'll close this issue

**Comment by sanjaypoyzer** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Can iPad / non-mouse users use the service without assistance?](https://trello.com/c/xTlU7rN1/3-can-ipad-non-mouse-users-use-the-service-without-assistance)
